org.cataclysmdda.CataclysmDDA.yml

```
@@ -0,0 +1,41 @@
+id: "org.cataclysmdda.CataclysmDDA"
+runtime: "org.freedesktop.Platform"
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: launch-cataclysm-dda
+finish-args: 
+  - --filesystem=xdg-config/cataclysm-dda:create
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - name: cataclysm-dda
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - mkdir -p /app/cataclysm-dda
+      - cp -r * /app/cataclysm-dda
+      - install -Dm755 launch-cataclysm-dda /app/bin/launch-cataclysm-dda
+      - install -Dm644 org.cataclysmdda.CataclysmDDA.svg /app/share/icons/hicolor/128x128/apps/org.cataclysmdda.CataclysmDDA.svg
```

```suggestion
      - install -Dm644 org.cataclysmdda.CataclysmDDA.svg /app/share/icons/hicolor/scalable/apps/org.cataclysmdda.CataclysmDDA.svg
```

Why is svg icons going in 128x128 ?